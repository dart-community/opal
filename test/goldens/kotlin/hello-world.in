const val MESSAGE = "Hello, world!"

fun main() {
    // Print the message.
    println("The message is: $MESSAGE")
}
