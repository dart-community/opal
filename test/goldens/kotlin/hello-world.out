const val MESSAGE = "Hello, world!"
^^^^^
source-code-kotlin-keyword-modifier-const
     ^
     source-code-kotlin-whitespace
      ^^^
      source-code-kotlin-keyword-declaration-val
         ^
         source-code-kotlin-whitespace
          ^^^^^^^
          source-code-kotlin-identifier-type
                 ^
                 source-code-kotlin-whitespace
                  ^
                  source-code-kotlin-keyword-operator
                   ^
                   source-code-kotlin-whitespace
                    ^
                    source-code-kotlin-literal-string-quoted-double-literal-string-quoted-double-begin
                     ^^^^^^^^^^^^^
                     source-code-kotlin-literal-string-quoted-double-literal-string-unquoted-literal-string-unquoted
                                  ^
                                  source-code-kotlin-literal-string-quoted-double-literal-string-quoted-double-end

fun main() {
^^^
source-code-kotlin-keyword-declaration-fun
   ^
   source-code-kotlin-whitespace
    ^^^^
    source-code-kotlin-identifier-function
        ^
        source-code-kotlin-punctuation
         ^
         source-code-kotlin-punctuation
          ^
          source-code-kotlin-whitespace
           ^
           source-code-kotlin-punctuation
    // Print the message.
^^^^
source-code-kotlin-whitespace
    ^^^^^^^^^^^^^^^^^^^^^
    source-code-kotlin-comment-line
    println("The message is: $MESSAGE")
^^^^
source-code-kotlin-whitespace
    ^^^^^^^
    source-code-kotlin-identifier-function
           ^
           source-code-kotlin-punctuation
            ^
            source-code-kotlin-literal-string-quoted-double-literal-string-quoted-double-begin
             ^^^^^^^^^^^^^^^^
             source-code-kotlin-literal-string-quoted-double-literal-string-unquoted-literal-string-unquoted
                             ^^^^^^^^
                             source-code-kotlin-literal-string-quoted-double-literal-string-unquoted-literal-string-interpolation
                                     ^
                                     source-code-kotlin-literal-string-quoted-double-literal-string-quoted-double-end
                                      ^
                                      source-code-kotlin-punctuation
}
^
source-code-kotlin-punctuation
